cocotb_build_libs.py

```diff
@@ -137,7 +119,7 @@ def _gen_import_libs(self, def_dir):
                 )
 
 
-def _extra_link_args(lib_name=None, rpath=None):
+def _extra_link_args(lib_name=None, rpaths=[]):
```

Comment:
```suggestion
def _extra_link_args(lib_name=None, rpaths=None):
```
I guess we should avoid `[]` here see: https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments

Reply:
You are right doe not really apply here.